core/vector.h

```diff
@@ -152,32 +153,34 @@ class Vector {
 
 template <class T>
 void Vector<T>::invert() {
+	int len = _cowdata->size();
+	int half_len = len / 2;
+	T *p = _cowdata->ptrw();
 
-	for (int i = 0; i < size() / 2; i++) {
-		T *p = ptrw();
-		SWAP(p[i], p[size() - i - 1]);
+	for (int i = 0; i < half_len; i++) {
+		SWAP(p[i], p[len - i - 1]);
 	}
 }
 
 template <class T>
 void Vector<T>::append_array(const Vector<T> &p_other) {
-	const int ds = p_other.size();
-	if (ds == 0)
+	const int other_len = p_other._cowdata->size();
+	if (other_len == 0)
 		return;
-	const int bs = size();
-	resize(bs + ds);
-	for (int i = 0; i < ds; ++i)
-		ptrw()[bs + i] = p_other[i];
+
+	const int len = _cowdata->size();
+	resize(len + other_len);
+	for (int i = 0; i < other_len; ++i)
+		_cowdata->ptrw()[len + i] = p_other[i];
 }
 
 template <class T>
 bool Vector<T>::push_back(const T &p_elem) {
-
-	Error err = resize(size() + 1);
+	Error err = resize(_cowdata->size() + 1);
 	ERR_FAIL_COND_V(err, true)
-	set(size() - 1, p_elem);
+	set(_cowdata->size() - 1, p_elem);
 
 	return false;
 }
 
-#endif
+#endif /* VECTOR_H */
```

Comment:
Yeah some contributors have IDEs that use a different style (the `/* FILENAME_H_ */` style is maybe from @hpvb's Eclipse?). But the majority are of the form `// FILENAME_H`, so that's what I try to enforce little by little. If that can be configured somehow in `.editorconfig`, that would be nice.